e2e/app/aou-elements/select-component.ts

```diff
@@ -42,31 +47,34 @@ export default class SelectComponent {
   }
 
   private async toggleOpenClose(): Promise<void> {
-    const selector = this.componentXpath() + '/*[@class="p-dropdown-trigger"]';
+    const selector = this.dropdownXpath() + '/*[@class="p-dropdown-trigger"]';
     const dropdownTrigger = await this.page.waitForXPath(selector, { visible: true });
     await dropdownTrigger.hover();
     await dropdownTrigger.click();
     await dropdownTrigger.dispose();
   }
 
   private async isOpen() {
-    const selector = this.componentXpath() + '/*[contains(normalize-space(@class),"p-dropdown-panel")]';
+    const selector = this.dropdownXpath() +
+       '/*[contains(concat(" ", normalize-space(@class), " "), " p-dropdown-panel ")]';
     const panel = await this.page.waitForXPath(selector);
     const classNameString = await (await panel.getProperty('className')).jsonValue();
     const splits = classNameString.toString().split(' ');
     await panel.dispose();
     return splits.includes('p-input-overlay-visible');
   }
 
-  private componentXpath(): string {
+  private dropdownXpath(): string {
     if (this.label === undefined) {
-      return '//*[contains(normalize-space(@class),"p-dropdown")]';
+      return '//*[contains(concat(" ", normalize-space(@class), " "), "p-dropdown")]';
```

Comment:
`class` can have multiple values and with similar strings. 
e.g a `div` element has`class="p-dropdown p-component"`. another `div` element has `class=p-dropdown-label`. Appending whitespaces before and after class values so that xpath won't match `p-dropdown-label`.